src/main/java/cloud/fogbow/common/constants/CloudStackConstants.java

```diff
@@ -5,6 +5,7 @@
 
     public static final String ERROR_CODE_KEY_JSON = "errorcode";
     public static final String ERROR_TEXT_KEY_JSON = "errortext";
+    public static final String JOB_ID_KEY_JSON = "jobid";
```

Comment:
Remove the empty line between the constants and put them in alphabetical order.

Reply:
Empty line removed.
About the Alphabetical order, I will talk to you in the chat.